src/service.ts

```diff
@@ -1213,6 +1247,14 @@ export class UserService extends ServiceBase<UserListResponse, UserList> impleme
         return returnOperationStatus(401, 'password does not match');
       }
 
+      await this.checkPasswordStrength(user.password).then((result) => {
```

Comment:
Make this check for password strength only if the `user.password` exists, currently this check is made even if password does not exist and will return an error for user update operation. Add this check at all 3 places.